bs-react-native-next/src/apis/TimePickerAndroid.md

```diff
@@ -0,0 +1,39 @@
+---
+id: apis/TimePickerAndroid
+title: TimePickerAndroid
+wip: true
+---
+
+```reason
+type options;
+
+[@bs.obj]
+external options:
+  (
+    ~hour: int=?,
+    ~minute: int=?,
+    ~is24Hour: bool=?,
+    ~mode: [@bs.string] [ | `clock | `spinner | `default ]=?
+    unit
+  ) =>
+  options =
+  "";
+
+type response = {
+  .
+  action: string,
+  hour: int,
+  minute: int
+};
+
+
+[@bs.module "react-native"] [@bs.scope "TimePickerAndroid"]
+external _open: options => Js.Promise.t(response) = "open";
```

Comment:
I guess you want `open_` here, because it is not unused and not only used internally.

Reply:
I thought `open_` and `_open` were equally fine, but probably not, see #403.